src/program/lwaftr/query/query.lua

```diff
@@ -143,8 +142,10 @@ function run (raw_args)
       end
    end
    if not pid then
+      print("pid, arg1, arg2", pid, arg1, arg2)
       top.select_snabb_instance(pid)
+      -- The following is not reached when there are multiple instances.
+      fatal("Please manually specify a pid, or a name with -n name")
```

Comment:
`top.select_snabb_instance` already does that check, I think it's not necessary.

https://github.com/Igalia/snabb/blob/lwaftr/src/program/top/top.lua#L34